### PR TITLE
Add periodic job for testing airship deploys

### PR DESCRIPTION
**Pull request by jesusaurus**
Add a copy of cloud-ccp-full-periodic that exercises the
airship deployment path.

### CODE REVIEW

File: jenkins/ci.suse.de/pipelines/cloud-ccp-full-periodic-airship.Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      steps {
+        sh "./run.sh enroll_caasp_workers"
+      }
+    }
+
+    stage('Deploy Airship') {
+      steps {
+        sh('''
+          USER=${USER:-root}  # why isn't this set?
+          ./run.sh setup_airship
```
Comment (dirkmueller) [inline jenkins/ci.suse.de/pipelines/cloud-ccp-full-periodic-airship.Jenkinsfile]: does this step deploy also openstack helm or is this separate? I think what w eneed to do is deploy airship, create the site manifest and then let airship deploy that site manifest which should deploy the openstack helm charts

**Comment by toabctl** [comment]
I think https://github.com/SUSE-Cloud/socok8s/pull/333 is the way to go...

File: jenkins/ci.suse.de/pipelines/cloud-ccp-full-periodic-airship.Jenkinsfile (hunk 1 above)
Comment (jesusaurus) [reply jenkins/ci.suse.de/pipelines/cloud-ccp-full-periodic-airship.Jenkinsfile]: Yes, this step does everything needed to deploy openstack using airship

**Comment by toabctl** [comment]
An alternative which adds PR testing for airship; https://github.com/SUSE-Cloud/automation/pull/3433 

There seems to be a jenkins problem (leading to a stacktrace) with this job definition. I'm in contact with @jdsn . Hopefully we can solve this.

File: jenkins/ci.suse.de/cloud-socok8s.yaml [hunk 2]
```diff
@@ -6,4 +6,5 @@
     repo-credentials: c2350527-476a-45df-b406-84f028614682
     jobs:
       - '{name}-integration'
+      - '{name}-nightly-trigger'
```
Comment (JanZerebecki) [inline jenkins/ci.suse.de/cloud-socok8s.yaml]: Why not one job for each deployment instead of a multijob?

Comment (JanZerebecki) [reply jenkins/ci.suse.de/cloud-socok8s.yaml]: My question is already outdated by https://github.com/SUSE-Cloud/automation/pull/3429 . Should this PR be closed?

Comment (jesusaurus) [reply jenkins/ci.suse.de/cloud-socok8s.yaml]: I don't think it makes sense to have two different jobs where the only difference is the default value of the deployment parameter.

**Pull request closed without merging**